roadmap.md

```diff
@@ -81,20 +82,25 @@ At Microsoft, .NET has always been an important component of other teams' produc
 
 |Milestone|Release Date|
```

Comment:
I'm wondering we should put the release-specific stuff (dates, focus) at the top. I'm "Mr. wordy" so I'm the one who wrote all those goals and stuff, but it seems like a lot of text to get through to get to the real roadmap.

This doc was intended as a mixture of a high-level contributor document and roadmap. I think the roadmap stuff should come first.